src/services/balancer/subgraph/entities/pools/index.ts

```diff
@@ -125,7 +133,8 @@ export default class Pools {
     if (hasLiquidityMiningRewards) {
       liquidityMiningAPY = computeAPYForPool(
         liquidityMiningRewards,
-        prices[TOKENS.AddressMap.BAL].price || 0,
+        prices[getOriginalAddress(this.networkId, TOKENS.AddressMap.BAL)]
```

Comment:
If I understood this function correctly it will map the original Mainnet addresses to the correct network address tokens. (assuming its defined)